settings.php

```diff
@@ -41,4 +41,8 @@
     $settings->add(new admin_setting_configcheckbox('logstore_xapi/backgroundmode',
         get_string('backgroundmode', 'logstore_xapi'),
         get_string('backgroundmode_desc', 'logstore_xapi'), 0));
+
+    $settings->add(new admin_setting_configtext('logstore_xapi/maxbatchsize',
```

Comment:
Yeah, I agree with @ryansmith94. I think it should be less than 500 as default. Our largest customer only does about 4,000 statements in a day, and I suspect many more would be a lot lower than that.